is_windy.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+'''
+Program to create dataframe with 'wind' column that is True when rain >0
+also creates a dataframe 'changes' that keeps track of how often this column changes
+Plots these columns.
+
+
+<input_data>   for example "foo.h5"
+<number_of_steps>  the number of data points before the rain column updates
+<start_time>  specify start of time interval
+<end_time>  specify end of time interval
+
+Usage:
+  is_windy.py <input_data> <number_of_steps> <start_time> <end_time>
+
+
+Options:
+  -h --help     Show this screen.
+  --version     Show version.
+'''
+
+
+
+import sys
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from astropy.table import Table
+import astropy
+from docopt import docopt
+from matplotlib import get_backend
+from matplotlib import interactive
+get_backend()
+interactive(True)
+
+def get_data(input_data ):
+    '''get data and clean it up for operation.
+    resample it so that there is one data point per minute.
+    '''
+
+    stuff = pd.read_hdf(input_data)
+    stuff['good_time'] = pd.to_datetime(stuff.Time*24*60*60*1e9)
+    stuff.set_index('good_time', inplace=True)
+    stuff.sort_index(inplace=True)
+    del stuff['QoS']
+    del stuff['Time']
+    del stuff['stat']
+    return stuff
+
+
+
+#Get data, convert time
+
+def add_column( data, number_of_steps ):
+
+    count_winds = []
+    count_calms = []
+    count_windy = 0
+    count_calm = 0
+    winds = []
+
+
+    for wind in data['v']:
+        if wind > 50:
```

Comment:
Could be nice if this limit could be a parameter to this function, this opens up possibilities to provide this limit from ... the command line .. or from a configuration file .. or may just from a `CONSTANT` which is defined in one of the first few lines of this script for simplicity.